exporter/stats/stackdriver/stackdriver.go

```diff
@@ -309,3 +312,24 @@ func namespacedViewName(v string, escaped bool) string {
 	}
 	return path.Join("custom.googleapis.com", p)
 }
+
+func newLabels(tags []tag.Tag) map[string]string {
+	labels := make(map[string]string)
+	for _, tag := range tags {
+		labels[tag.Key.Name()] = tag.Value
+	}
+	return labels
+}
+
+func newLabelDescriptors(keys []tag.Key) []*labelpb.LabelDescriptor {
+	labelDescriptors := make([]*labelpb.LabelDescriptor, len(keys))
+	for _, key := range keys {
+		labelDescriptors = append(
+			labelDescriptors,
+			&labelpb.LabelDescriptor{
+				Key:       key.Name(),
```

Comment:
The tag keys will need to be escaped for Stackdriver.  It limits length to 100 and only accepts some characters.  Letters, digits and underscores work.

Reply:
Sure, I added string sanitization in https://github.com/census-instrumentation/opencensus-go/pull/167